tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -1298,13 +1302,13 @@ internal void CollectTelemetry(IConfigurationTelemetry destination)
 
                 if (supportedSamplerName is null)
                 {
-                    // TODO log warning that the OpenTelemetry value is invalid
+                    log.LogUnsupportedConfiguration(otelSampleType.Key, samplerName, ddSampleRate.Key);
```

Comment:
In this case, the Datadog key was not set (since it is not present). Should we do this instead?

```suggestion
                    log.LogInvalidConfiguration(otelSampleType.Key);
```

Reply:
Good question 🤔 The thing is, it's not an _invalid_ configuration necessarily, just one we don't support (yet)? Maybe we should just do a slightly different wording here?

```suggestion
                    log.EnqueueAction(
                        (log, _) =>
                        {
                            log.Warning(
                                "OpenTelemetry configuration {OpenTelemetryConfiguration}={OpenTelemetryValue} is not supported.",
                                otelSampleType.Key,
                                samplerName);
                        });
```